skrifa/src/meta/strings.rs

```diff
@@ -0,0 +1,683 @@
+    (0x1c0a, "es-DO"),        //Spanish
+    (0x1c1a, "sr-Cyrl-BA"),   //Serbian (Cyrillic)
+    (0x1c3b, "sma-SE"),       //Sami (Southern)
+    (0x1c3b, "sma"),          //Sami (Southern)
+    (0x2001, "ar-OM"),        //Arabic
+    (0x2009, "en-JM"),        //English
+    (0x200a, "es-VE"),        //Spanish
+    (0x201a, "bs-Cyrl-BA"),   //Bosnian (Cyrillic)
+    (0x201a, "bs-Cyrl"),      //Bosnian (Cyrillic)
+    (0x203b, "sms-FI"),       //Sami (Skolt)
+    (0x203b, "sms"),          //Sami (Skolt)
+    (0x2401, "ar-YE"),        //Arabic
+    (0x2409, "en-029"),       //English
+    (0x240a, "es-CO"),        //Spanish
+    (0x241a, "sr-Latn-RS"),   //Serbian (Latin)
+    (0x243b, "smn-FI"),       //Sami (Inari)
+    (0x2801, "ar-SY"),        //Arabic
+    (0x2809, "en-BZ"),        //English
+    (0x280a, "es-PE"),        //Spanish
+    (0x281a, "sr-Cyrl-RS"),   //Serbian (Cyrillic)
+    (0x2c01, "ar-JO"),        //Arabic
+    (0x2c09, "en-TT"),        //English
+    (0x2c0a, "es-AR"),        //Spanish
+    (0x2c1a, "sr-Latn-ME"),   //Serbian (Latin)
+    (0x3001, "ar-LB"),        //Arabic
+    (0x3009, "en-ZW"),        //English
+    (0x300a, "es-EC"),        //Spanish
+    (0x301a, "sr-Cyrl-ME"),   //Serbian (Cyrillic)
+    (0x3401, "ar-KW"),        //Arabic
+    (0x3409, "en-PH"),        //English
+    (0x340a, "es-CL"),        //Spanish
+    (0x3801, "ar-AE"),        //Arabic
+    (0x380a, "es-UY"),        //Spanish
+    (0x3c01, "ar-BH"),        //Arabic
+    (0x3c0a, "es-PY"),        //Spanish
+    (0x4001, "ar-QA"),        //Arabic
+    (0x4009, "en-IN"),        //English
+    (0x400a, "es-BO"),        //Spanish
+    (0x4409, "en-MY"),        //English
+    (0x440a, "es-SV"),        //Spanish
+    (0x4809, "en-SG"),        //English
+    (0x480a, "es-HN"),        //Spanish
+    (0x4c0a, "es-NI"),        //Spanish
+    (0x500a, "es-PR"),        //Spanish
+    (0x540a, "es-US"),        //Spanish
+];
+
+#[cfg(test)]
+mod tests {
+    use crate::MetadataProvider;
+
+    use super::*;
+    use read_fonts::FontRef;
+
+    #[test]
+    fn string_ids() {
+        let font = FontRef::new(read_fonts::test_data::test_fonts::VAZIRMATN_VAR).unwrap();
+        assert_eq!(
+            font.string_ids().collect::<Vec<_>>().as_slice(),
+            &[
+                StringId::SUBFAMILY_NAME,
+                StringId::new(257),
+                StringId::new(258),
+                StringId::new(261),
+                StringId::new(264),
+            ]
+        );
+    }
+
+    #[test]
+    fn localized() {
+        let font = FontRef::new(read_fonts::test_data::test_fonts::NAMES_ONLY).unwrap();
+        let mut subfamily_names = font
+            .localized_strings(StringId::SUBFAMILY_NAME)
+            .map(|s| (s.language().unwrap().to_string(), s.to_string()))
+            .collect::<Vec<_>>();
+        subfamily_names.sort_by(|a, b| a.0.cmp(&b.0));
+        let expected = [
+            (String::from("el-GR"), String::from("Κανονικά")),
```

Comment:
Could we test a non-European language for a result here as well, Arabic and Chinese in particular? (the intent being to cover the UTF-8 conversion handling.)

Reply:
Absolutely. I’ll add subfamily names in those languages.